OrbitLinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -16,13 +16,16 @@ void UprobesUnwindingVisitor::visit(SamplePerfEvent* event) {
   const std::vector<unwindstack::FrameData>& full_callstack =
       unwinder_.Unwind(current_maps_.get(), event->GetRegisters(),
                        event->GetStackData(), event->GetStackSize());
-  if (!full_callstack.empty()) {
+  // Some samples can actually fall inside u(ret)probes code. Discard them,
+  // because when they are unwound successfully the result is wrong.
+  if (!full_callstack.empty() &&
+      full_callstack.front().map_name != "[uprobes]") {
```

Comment:
As mentioned in the other comment, we should count how often we discard a sample due to falling into u(ret)probe code area.

Reply:
Done, see other comment.